### PR TITLE
Update travisCI

### LINKED ISSUE #3: Fix travisCI
- [x] Delete `before install` from travisCI script
> these package are installed from the requirements
- [x] Moving linters and misc checks into jobs via jobs.include

### PR DESCRIPTION
**Pull request by PetrushynskyiOleksii**
Resolve #3

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -2,11 +2,11 @@ language: python
...
-- pip install django==1.11
-
-script:
-- pre-commit run --all-files
-- python ./manage.py test
+jobs:
+  include:
+    - stage: pre-commit tests
+      script:
+        - pre-commit run --all-files
```
Comment (webknjaz) [inline .travis.yml]: Dedent sequence items

File: .travis.yml [hunk 2]
```diff
@@ -2,11 +2,11 @@ language: python
...
+    - stage: django tests
```
Comment (webknjaz) [inline .travis.yml]: You don't have to move main tests to jobs

**Review by webknjaz — CHANGES_REQUESTED** [review]

File: .travis.yml (hunk 2 above)
Comment (PetrushynskyiOleksii) [reply .travis.yml]:
```
script:
- python ./manage.py test

jobs:
  include:
    - stage: pre-commit tests
      script:
      - pre-commit run --all-files
```
Like this? Or how? I do not quite understand, @webknjaz .

Comment (webknjaz) [reply .travis.yml]: Yep

File: .travis.yml [hunk 3]
```diff
@@ -2,11 +2,11 @@ language: python
...
-- pip install -U pre-commit
-- pip install django==1.11
-
 script:
-- pre-commit run --all-files
 - python ./manage.py test
+
+jobs:
+  include:
+    - stage: pre-commit tests
```
Comment (webknjaz) [inline .travis.yml:10]: @PetrushynskyiOleksii this is also a sequence item to dedent

Comment (PetrushynskyiOleksii) [reply .travis.yml:10]: ok, i'll correct it

**Pull request merged**